builder/parallels/common/driver.go

```diff
@@ -78,23 +82,20 @@ func NewDriver() (Driver, error) {
 			"Parallels builder works only on \"darwin\" platform!")
 	}
 
-	cmd := exec.Command("/usr/bin/python3", "-c", `import prlsdkapi`)
-	err := cmd.Run()
-	if err != nil {
-		return nil, fmt.Errorf(
-			"Parallels Virtualization SDK is not installed")
-	}
-
 	if prlctlPath == "" {
 		var err error
 		prlctlPath, err = exec.LookPath("prlctl")
 		if err != nil {
 			return nil, err
 		}
 	}
-
 	log.Printf("prlctl path: %s", prlctlPath)
 
+	err := checkforPythonSDK(prlctlPath)
```

Comment:
when defining an error as an output and an if for that error the most common way of doing it is
if err := checkforPythonSDK(prlctlPath); err != nil {
  return nil, err
}
but you don't need to change it, just easier to read

Reply:
I will adapt it for my next changes. I should learn Go way of writing :). I did it as a C++ programer. Thanks for mentioning @cjlapao